app/src/main/java/com/example/mensfashion/ui/details/DetailsFragment.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.mensfashion.ui.details
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.mensfashion.R
+
+
+class DetailsFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+    }
+
+    override fun onCreateView(
```

Comment:
you don't need onCreateView any more

Reply:
yes, it was  an initial code, I forget to remove it after using Basefragment on the other screen